src/main/java/ch/njol/skript/aliases/ItemType.java

```diff
@@ -360,6 +362,23 @@ public boolean setBlock(Block block, boolean applyPhysics) {
 		return false;
 	}
 	
+	/**
+	 * Send a block change to a player
+	 * <p>This will send a fake block change to the player, and will not change the block on the server.</p>
+	 *
+	 * @param player Player to send change to
+	 * @param location Location of block to change
+	 */
+	public void sendBlockChange(Player player, Location location) {
+		for (int i = random.nextInt(types.size()); i < types.size(); i++) {
```

Comment:
Out of curiosity, why do you use random here?

Reply:
the code was just copied over from Eff class.

Reply:
it looks like it's from ItemType#setBlock. Since that method is in ItemType, shouldn't this be to and then you just call it here?